Tools/_sunrise/changelog/actions_changelogs_since_last_run.py

```
@@ -525,8 +675,31 @@ def diff_changelog(
     """
     Находит новые записи, которых не было в предыдущей публикации.
     """
-    old_entry_ids = {e["id"] for e in old["Entries"]}
-    return (e for e in cur["Entries"] if e["id"] not in old_entry_ids)
+    old_entries = old.get("Entries")
+    current_entries = cur.get("Entries")
+    if not isinstance(old_entries, list) or not isinstance(current_entries, list):
+        raise RuntimeError("Обе версии чейнжлога должны содержать список Entries")
```

Ruff (0.16.3)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)